tests/WorkOS/ClientTest.php

```diff
@@ -119,9 +119,9 @@ public function testClientThrowsRequestExceptionsWithMessageAndCode($statusCode,
         }
     }
 
-     /**
-     * @dataProvider requestExceptionTestProvider
-     */
+    /**
+    * @dataProvider requestExceptionTestProvider
```

Comment:
Looks like these the asterisks on the following lines might need to be indented slightly:

```
/**
 * @dataProvider requestExceptionTestProvider
 */
```

Reply:
@maxdeviant thanks - got those indented. Seems like the ol' linter only cared about the top line but now they're all uniform 👍